src/wmtk/Mesh.hpp

```diff
@@ -338,33 +338,166 @@ class Mesh : public std::enable_shared_from_this<Mesh>
     //============================
     // MultiMesh interface
     //============================
+    //
+    /**
+     * @brief return if this mesh is the root of a multimesh tree
+     */
     bool is_multi_mesh_root() const;
+    /**
+     * @brief returns a reference to the root of a multimesh tree
+     */
     Mesh& get_multi_mesh_root();
-    std::vector<std::shared_ptr<Mesh>> get_child_meshes() const;
+    /**
+     * @brief returns a const reference to the root of a multimesh tree
+     */
     const Mesh& get_multi_mesh_root() const;
+
+    /**
+     * @brief returns the direct multimesh child meshes for the current mesh
+     */
+    std::vector<std::shared_ptr<Mesh>> get_child_meshes() const;
+
+    /**
+     * @brief returns a unique identifier for this mesh within a single multimesh structure
```

Comment:
Maybe add a comment what the id looks like. It might be surprising that the id is a whole vector of longs.